lib/orbit-common/cache.js

```diff
@@ -490,7 +505,7 @@ var Cache = Class.extend({
     }
   },
 
-  _transformRelatedInverseLinks: function(operation) {
+  _transformRelatedInverseLinkOps: function(operation) {
```

Comment:
These renames, adding `Ops` no longer directly transform should they drop the transform prefix?